client-runtime/src/main/java/com/microsoft/rest/v2/SwaggerMethodProxyDetails.java

```diff
@@ -233,7 +233,7 @@ public void setReturnType(Class<?> returnType) {
      * object that is returned from the method.
      * @return The synchronous return type for the method that this object describes.
      */
-    public Class<?> getReturnType() {
+    public Class<?> returnType() {
```

Comment:
We may need to use Type instead of Class<?> since Class<?> has difficulty handling generic types. See https://github.com/Azure/autorest-clientruntime-for-java/blob/master/client-runtime/src/main/java/com/microsoft/rest/serializer/JacksonAdapter.java#L127.